### PR TITLE
Add Contrib to the Ajax paths. #1519

### PR DESCRIPTION
**Pull request by dpvc**
Add `[Contrib]` to the Ajax paths.  Resolves issue #1519.

### CODE REVIEW

**Comment by pkra** [comment]
LGTM.

Should we consider enforcing `https` in v3.0? Right now, this would break very old systems like WinXP with outdated cert infrastructures but perhaps it is time for 3.0.

**Comment by dpvc** [comment]
Is there anything really that needs to be done differently for `https` from `http`?  If the only accommodation that needs to be made is to check the protocol now and then, I don't see the harm in doing so.  But I don't have any strong feelings about it.

**Comment by pkra** [comment]
> If the only accommodation that needs to be made is to check the protocol now and then, I don't see the harm in doing so.

Right. I suppose if the CDN always loaded via https, then at least it can't be a point of attack. Which, arguably, is very little if the site is on http.

> But I don't have any strong feelings about it.

Me neither.

But now I spotted a typo!

File: unpacked/MathJax.js [hunk 1]
```diff
@@ -659,6 +659,8 @@ MathJax.cdnFileVersions = {};  // can be used to specify revisions for individua
   
   var PATH = {};
   PATH[BASENAME] = "";  // empty path gets the root URL
+  PATH.Contrib = (String(location.protocal).match(/^https?:/) ? "" : "http:") +
```
Comment (pkra) [inline unpacked/MathJax.js]: `location.protocol`?

Comment (pkra) [inline unpacked/MathJax.js]: Sorry for not catching that before.

Comment (dpvc) [inline unpacked/MathJax.js]: Argh!  I'll fix it.

**Pull request merged**